### PR TITLE
Add support for CLUSTER LINKS command

### LINKED ISSUE #2386: Add support for CLUSTER LINKS command
Redis CLUSTER LINKS command was added in version 7.0.0 and is currently supported by the library. We should add support as documented [here](https://redis.io/commands/CLUSTER-LINKS).

### PR DESCRIPTION
**Pull request by SoulPancake**
Fixes #2386

### CODE REVIEW

**Comment by SoulPancake** [comment]
@monkey92t Can you please review this when you have time

File: command.go [hunk 1]
```diff
@@ -4246,3 +4246,96 @@ func (cmd *KeyFlagsCmd) readReply(rd *proto.Reader) error {
...
+	}
+	cmd.val = make([]ClusterLink, n)
+
+	for i := 0; i < len(cmd.val); i++ {
+		m, err := rd.ReadMapLen()
+		if err != nil {
+			return err
+		}
+
+		link := ClusterLink{}
```
Comment (monkey92t) [inline command.go:4307]: duplicate memory allocation, `make([]ClusterLink)` has allocated memory for ClusterLink.

File: command.go [hunk 2]
```diff
@@ -4246,3 +4246,96 @@ func (cmd *KeyFlagsCmd) readReply(rd *proto.Reader) error {
...
+			case "create-time":
+				link.CreateTime, err = rd.ReadInt()
+			case "events":
+				link.Events, err = rd.ReadString()
+			case "send-buffer-allocated":
+				link.SendBufferAllocated, err = rd.ReadInt()
+			case "send-buffer-used":
+				link.SendBufferUsed, err = rd.ReadInt()
+			default:
+				return fmt.Errorf("unsupported key in cluster link info: %s", key)
```
Comment (monkey92t) [inline command.go:4329]: format: `redis: xxxx`

**Review by monkey92t — APPROVED** [review]
tks...

**Pull request merged**